packages/cluster/src/commands/db_cmds/restore.js

```diff
@@ -20,7 +20,7 @@ const run = async function(argv) {
         process.exit(1)
     }
 
-    const rest = await tryCatchAsync(
+    const res = await tryCatchAsync(
```

Comment:
Lint rule: no unused variables

Also a bug where `res` is accessed later, but the variable is named `rest`.